aspnetcore/fundamentals/url-rewriting.md

```diff
@@ -96,47 +112,75 @@ Original Request using `AddRedirectToHttpsPermanent()`: `/secure`
 
-There is no roundtrip to the server to obtain the resource. If the resource exists, it's fetched and returned to the client with a 200 (OK) status code. Because the client isn't redirected, the URL in the browser address bar doesn't change. As far as the client is concerned, the URL rewrite operation never occurred.
+There's no roundtrip to the server to obtain the resource. If the resource exists, it's fetched and returned to the client with a 200 (OK) status code. Because the client isn't redirected, the URL in the browser address bar doesn't change. As far as the client is concerned, the URL rewrite operation never occurred.
 
 > [!NOTE]
-> `skipRemainingRules: true` should be used whenever possible, because matching rules is an expensive process and slows down application response time. For the fastest application response, order your rewrite rules from most frequently matched to least frequently matched and skip the processing of the remaining rules when a match occurs and no additional rule processing is required.
+> Use `skipRemainingRules: true` whenever possible, because matching rules is an expensive process and reduces app response time. For the fastest app response, order your rewrite rules from the most frequently matched rule to the least frequently matched rule and skip the processing of the remaining rules when a match occurs and no additional rule processing is required.
 
 ### Apache mod_rewrite
-You can apply Apache mod_rewrite rules with `AddApacheModRewrite()`. The first parameter takes an `IFileProvider`, which is provided in the sample application via [Dependency Injection](dependency-injection.md) by injecting the `IHostingEnvironment` and using it to provide the `ContentRootFileProvider`. The second parameter is the path to your rules file, which is *ApacheModRewrite.txt* in the sample application. You must make sure that the rules file is deployed with the application. For more information and examples of mod_rewrite rules, see [Apache mod_rewrite](https://httpd.apache.org/docs/2.4/rewrite/).
+Apply Apache mod_rewrite rules with `AddApacheModRewrite()`. Make sure that the rules file is deployed with the app. For more information and examples of mod_rewrite rules, see [Apache mod_rewrite](https://httpd.apache.org/docs/2.4/rewrite/).
+
+# [ASP.NET Core 1.x](#tab/aspnetcore1x)
+
+The first parameter takes an `IFileProvider`, which is provided in the sample app via [Dependency Injection](dependency-injection.md) by injecting the `IHostingEnvironment` and using it to provide the `ContentRootFileProvider`. The second parameter is the path to your rules file, which is *ApacheModRewrite.txt* in the sample app.
```

Comment:
The first parameter takes an `IFileProvider`, which is provided in the sample app via [Dependency Injection](dependency-injection.md) by injecting the `IHostingEnvironment` and using it to provide the `ContentRootFileProvider`. 

2 or more sentences